x-pack/plugins/maps/public/actions/store_actions.js

```diff
@@ -156,12 +156,13 @@ export function addLayer(layerDescriptor) {
   };
 }
 
-export function setLayerErrorStatus(layerId, errorMessage) {
+function setLayerDataLoadErrorStatus(layerId, errorMessage) {
   return dispatch => {
     dispatch({
       type: SET_LAYER_ERROR_STATUS,
+      isInErrorState: errorMessage !== null ? true : false,
```

Comment:
This can just be `errorMessage !== null`, there is no need for the ternary operator